[README.md] New `Exploring swift-java` and `troubleshoot` sections (#21)
swiftlang/swift-android-examples: hello-swift-java/README.md

    ./gradlew :hello-swift-java-hashing-lib:assembleRelease
    ```

2.  After a successful build, the Android library will be located at `swift-java-hashing-example/hashing-lib/build/outputs/aar/hashing-lib-release.aar`.

## Exploring `swift-java`
We encourage exploring `swift-java` by writing Swift code to the file `hello-swift-java/hashing-lib/Sources/SwiftHashing/SwiftHashing.swift` and building the `hashing-app` target.\
> [!TIP]
> The list of supported features can be found at:\
> https://swiftpackageindex.com/swiftlang/swift-java/main/documentation/swiftjavadocumentation/supportedfeatures#JExtract-calling-Swift-from-Java\

Once the `hashing-app` build completes, the newly generated Java code will be available to use in the Android project.

While exploring, some errors might occur. Following is a list of errors and how to troubleshoot them.

> >⚠️⚠️ These are temporary and will not be required once the gradle plugin is automated ⚠️⚠️

Reviewer: ```suggestion
## Troubleshooting
```

Reviewer 2: I'd keep a soft note, without the scary ⚠️ :)

Good to make it a new section tho 
```suggestion
## Troubleshooting

> Most of these issues are temporary and will be resolved in the future as the swift-java project matures and automates more of the build steps.

```

___
### Swift/Java not available for use once Android build completes:
If after adding new code to `SwiftHashing.swift` the same is not available on Android after a build, please delete the swift `.build` folder located at `hello-swift-java/hashing-lib/.build` (Attention to the dot before the folder name `.build`)

Build the Android `hashing-app` target again and the code should be avilable.

---

> ### Crash: Library not found

Reviewer: ```suggestion
#### Crash: Library not found
```
